portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,45 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
+/*
+ * Adds a random activity car to the page.
  */
-function addRandomGreeting() {
-  const greetings =
-      ['Gone Now by Bleachers', 'Malibu Nights by LANY', 'Melodrama by Lorde','After Hours by The Weeknd', 'American Teen by Khalid'];
+function addRandomActivity() {
+  const activityTitles = [
+    "Bake Banana Bread!",
```

Comment:
For here and everywhere below, we use single quotes for JavaScript strings instead of double.
Here's the style guide: https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes